clang/test/Driver/openmp-system-arch.c

```diff
@@ -1,4 +1,3 @@
-// REQUIRES: system-linux
```

Comment:
This is failing on z/OS on line 33 with the error:
```
clang: error: failed to deduce triple for target architecture 'native'; specify the triple using '-fopenmp-targets' and '-Xopenmp-target' instead
```
Not deducing the target gpu causes wrong -target-cpu to be specified.

If it can be fixed great.  If not can you add:
```
UNSUPPORTED: system-zos
```